src/thunderbots/software/radio_communication/mrf/dongle.cpp

```diff
@@ -284,23 +284,28 @@ void MRFDongle::handle_mdrs(AsyncOperation<void> &op)
 void MRFDongle::handle_message(AsyncOperation<void> &, USB::BulkInTransfer &transfer)
 {
     transfer.result();
-
+    bool to_beep = false;
     // Only handle if there are more than 2 bytes in the transfer.
     if (transfer.size() > 2)
     {
         unsigned int robot = transfer.data()[0];
-        annunciator.handle_robot_message(robot, transfer.data() + 1, transfer.size() - 3,
-                                         transfer.data()[transfer.size() - 2],
-                                         transfer.data()[transfer.size() - 1]);
+        to_beep            = annunciator.handle_robot_message(
+            robot, transfer.data() + 1, transfer.size() - 3,
+            transfer.data()[transfer.size() - 2], transfer.data()[transfer.size() - 1]);
     }
     transfer.submit();
+    if (to_beep)
+    {
+        beep(ANNUNCIATOR_BEEP_LENGTH_MILLISECONDS);
+    }
 }
 
 void MRFDongle::handle_status(AsyncOperation<void> &)
 {
     status_transfer.result();
     estop_state = static_cast<EStopState>(status_transfer.data()[0] & 3U);
-    annunciator.handle_status(status_transfer.data()[0U]);
+    // TODO: beep on thesemessages later?
+    annunciator.handle_dongle_messages(status_transfer.data()[0U]);
```

Comment:
TODO?

Reply:
Was thinking about beeping the dongle for this type of messages too, thought it would be better handled in a separate PR.

Reply:
Guess it wouldn't be too much work to stick that in here though...

Reply:
@MathewMacDougall looking at the messages handled by `handle_dongle_messages` (`annunciator.cpp:371`), would you agree that these messages are critical enough that the dongle should continuously beep while those messages are true?

Reply:
All good with doing this in another PR, let's just make an issue for it if that's the way we decide to go.

Reply:
@henrymliu 	I'd say the messages in `handle_dongle_messages` don't warrant continuous beeping. They're _relatively_ obvious, persistent, and easy to fix compared to critical robot issues.